modules/schema/src/category/table/table-types.ts

```diff
@@ -71,18 +71,18 @@ export type TableBatch = Batch & {
 
 /** Batch for a table organized as an array of rows */
 export type RowTableBatch = TableBatch & {
-  shape: 'row-table';
+  shape: 'row-table' | 'array-row-table' | 'object-row-table' | 'geojson-row-table';
```

Comment:
Can we search for `geojson-table` I think I used that in a few places? I am fine with either name, but it should be consistent.